mne/viz/_3d.py

```diff
@@ -598,11 +597,13 @@ def _limits_to_control_points(clim, stc_data, colormap):
 
 
 def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
-                          colormap='auto', time_label='time=%0.2f ms',
+                          colormap='auto', time_label='auto',
```

Comment:
Because the actual label is `time_label % time`, so without changing it, after plotting `stc.plot(time_unit='s')` the user would see "time=-0.01 ms" when the actual time is -100 ms (-0.1 s)...